creator-node/test/contentBlacklist.test.js

```diff
@@ -1183,7 +1183,7 @@ describe('test ContentBlacklist', function () {
     })
 
     // Return user and some track data
-    return { cnodeUser, ...trackUploadResp, sessionToken }
+    return { cnodeUser, ...trackUploadResp, trackId, sessionToken }
```

Comment:
why this change?

Reply:
got lint error that trackId was not consumed. i just returned it in the response for simplicity

Reply:
cool